dashboard/pages/correlation/callbacks.py

```diff
@@ -154,6 +152,17 @@ def on_visualization_stage_entry(
     )
     concentration_fig = concentration_plot(df, "INHIBITION")
 
+    report_data_correlation_plots = {
+        "concentration_value": concentration_value,
+        "volume_value": volume_value,
+        "inhibition_fig": inhibition_fig.to_html(
```

Comment:
At this point, as a result from the screening process, one may have either `% IHIBITION` or `% ACTIVATION` (not both). 
Now if one wants to examine the correlation between screenings containing `% ACTIVATION`, they'll get KeyError (bug 🪱).

To avoid this, could you add sth like :
```
feature = "% ACTIVTION" if "% ACTIVTION" in df.columns else "% INHIBITION"
concentration_fig = concentration_plot (df, feature) 
[...]
```
and then update the `report_data_correlation_plots`: ~~inhibition_fig~~ -> feature_fig

Reply:
It worked! Thanks for help 😁😍